requirements/frozen.txt

```diff
@@ -290,7 +290,7 @@ requests-oauthlib==1.2.0
 requests-toolbelt==0.9.1
 requestsexceptions==1.4.0
 rfc3986==1.2.0
-riggerlib==3.1.5
+git+https://github.com/jaryn/riggerlib.git#fix_no_listener_deadlock
```

Comment:
This is the reason for NOMERGE in title - once we have a riggerlib release for this we can merge.

The riggerlib update can come in another PR as well - we can just reset this to 3.1.5 for this PR's purpose.